karma.conf.js

```diff
@@ -0,0 +1,46 @@
+module.exports = function (config) {
+  config.set({
+
+    port: 9876,
+
+    logLevel: config.LOG_INFO,
+
+    autoWatch: true,
+    singleRun: false,
+    colors: true,
+
+    plugins: [
+      'karma-jasmine',
+      'karma-sinon',
+      'karma-spec-reporter',
+      'karma-chrome-launcher',
+      'karma-firefox-launcher',
+      'karma-browserify'
+    ],
+
+    browsers: ['Chrome'],
```

Comment:
As long as we at least run the tests in CLI by default, I'm content. I just didn't want to totally make it specific to just the browser when we run tests.